选修课程/计算机视觉/HITCV_EXP/exp3/exp-03-code/KAIR/utils/utils_model.py

```
@@ -0,0 +1,330 @@
+    msg += 'Net structure:\n{}'.format(str(model)) + '\n'
+    return msg
+
+
+# --------------------------------------------
+# parameters description
+# --------------------------------------------
+def describe_params(model):
+    if isinstance(model, torch.nn.DataParallel):
+        model = model.module
+    msg = '\n'
+    msg += ' | {:^6s} | {:^6s} | {:^6s} | {:^6s} || {:<20s}'.format('mean', 'min', 'max', 'std', 'shape', 'param_name') + '\n'
```

Ruff (0.15.21)

[error] `.format` call has unused arguments at position(s): 5

Remove extra positional arguments at position(s): 5

(F523)